documentation/authentication.md

```diff
@@ -3,11 +3,59 @@
 
 # Authentication
 
-Authenticated requests to `dotnet monitor` help protect sensitive diagnostic artifacts from unauthorized users and lower privileged processes. `dotnet monitor` can be configured to use either [Windows Authentication](#windows-authentication) or via an [API Key](#api-key-authentication). It is possible, although strongly not recommended, to [disable authentication](#disabling-authentication).
+Authenticated requests to `dotnet monitor` help protect sensitive diagnostic artifacts from unauthorized users and lower privileged processes. `dotnet monitor` can be configured to use any one of the following authentication modes:
+- [API Key](#api-key-authentication)
+- [Azure Active Directory Authentication](#azure-active-directory-authentication) (8.0+)
```

Comment:
Would this work/ would you be able to test it with an MSA account rather than the organizational account?